app/infra/models/memorial.py

```diff
@@ -13,8 +13,7 @@ def to_json(self, *args, **kwargs):
         return {
             "firstName": self.first_name,
             "lastName": self.last_name,
-            "description": self.description,
-            "online": self.online,
+            "description": self.description
```

Comment:
What was initially wrong with the self.online?

Reply:
It's redundant because we have a "view" permission. So if the "view" permission is set to "owner" it's effectively "offline" for everyone else